CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### API changes
+
+- App Hang Tracking for iOS is now disabled by default. You'll need to enable this manually if you want to use it in your applications. ([#4320](https://github.com/getsentry/sentry-dotnet/pull/4320))
```

Comment:
I imagine there's a reason for this. Would be nice to include it here or link to it